### DISCUSSION TITLE
Who can publish an action to the Marketplace

### DISCUSSION THREAD

**Discussion by jonasbn** [community/community#25403] [Actions]
<p>I have been added to a repository as contributor, but I cannot publish the releases to the Marketplace - what are the requirements apart from accepting the terms and conditions, which I have done via one of my own GitHub Action repositories.</p>
<p><a href="https://help.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace" rel="nofollow noopener">The documentation</a> does not mention any special requirements in this regard.</p>
<p>The repository in question is: <a href="https://github.com/rojopolis/spellcheck-github-actions" rel="nofollow noopener">rojopolis/spellcheck</a></p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/jonasbn">@jonasbn</a>,</p>
<p>Please add action.yml to your repository. I forked your repo, it works for me:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/95841a6d9643a67c22b13f56358643d3519c7394.png" data-download-href="/uploads/short-url/lkGaIU2hQC77Ihl3tFF5OxOjEaw.png?dl=1" title="publishtoMarketplace" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181090775-2efd95e9-09b2-45f2-93b3-b49c28399f53.png" alt="publishtoMarketplace" data-base62-sha1="lkGaIU2hQC77Ihl3tFF5OxOjEaw" width="587" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/9/95841a6d9643a67c22b13f56358643d3519c7394_2_587x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/9/95841a6d9643a67c22b13f56358643d3519c7394_2_880x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/9/95841a6d9643a67c22b13f56358643d3519c7394.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/95841a6d9643a67c22b13f56358643d3519c7394_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">publishtoMarketplace</span><span class="informations">941×801 53.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I confirmed contributor can publish action to marketplace as well.</p>
<p>Thanks</p>

**Comment by jonasbn** [comment]
<p>Hi <a class="mention" href="/weide-zhou">@weide-zhou</a></p>
<p>Thanks for you response.</p>
<p>Rereading the documentation I can see that this file is mentioned, but the action has previously been <a href="https://github.com/marketplace/actions/spellcheck-action" rel="nofollow noopener">published to the Marketplace successfully</a> without this particular file.</p>
<p>I will attempt your suggested approach, but something does not seem right and it means I cannot release a earlier release (0.3.0).</p>
<p>Only explanation I can come up with is that the requirements has changed over time.</p>

**Comment by jonasbn** [comment]
<p>Hi <a class="mention" href="/weide-zhou">@weide-zhou</a>,</p>
<p>This worked like a charm - thanks for taking the time to respond, it is really appreciated.</p>
<p>Take care and stay safe,</p>
<p>jonasbn</p>
